crates/turborepo-lib/src/hash/mod.rs

```diff
@@ -41,15 +41,16 @@ mod proto_capnp {
     }
 }
 
+#[derive(Debug)]
 pub struct TaskHashable<'a> {
     // hashes
     pub(crate) global_hash: &'a str,
     pub(crate) task_dependency_hashes: Vec<String>,
     pub(crate) hash_of_files: &'a str,
-    pub(crate) external_deps_hash: String,
+    pub(crate) external_deps_hash: Option<String>,
```

Comment:
We need to wrap fields in `Option` to mimic how Go automatically omits string fields that are empty when hashing to capnproto (it can do this because all fields are optional in capnproto).